modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Overview/index.js

```diff
@@ -34,73 +34,71 @@ const Overview = ({project, subscriptionGroups}) => {
 		SUBSCRIPTIONS_STATUS.future,
 	]);
 
+	const [allSubscriptions, setAllSubscriptions] = useState([]);
+
+	const [
+		subscriptionGroupsWithSubscriptions,
+		setSubscriptionGroupsWithSubscriptions,
+	] = useState([]);
+
 	const parseAccountSubscriptionGroupERC = (subscriptionName) => {
 		return subscriptionName.toLowerCase().replace(' ', '-');
 	};
 
-	const getAccountSubscriptionFilterQueryString = (
-		previousValue,
-		currentValue,
-		currentIndex,
-		array
-	) => {
-		if (currentIndex === array.length - 1) {
-			return previousValue + ` subscriptionStatus eq '${currentValue}'`;
-		}
-
-		return (
-			previousValue +
-			` subscriptionStatus eq '${currentValue}' or accountSubscriptionGroupERC eq '${
-				project.accountKey
-			}_${parseAccountSubscriptionGroupERC(
-				selectedSubscriptionGroup
-			)}' and`
-		);
-	};
+	useEffect(() => {
+		const subscriptionsToRender = allSubscriptions
```

Comment:
Use here the useMemo variable suggested below, example:
```suggestion
		const subscriptionsCards = accountSubscriptionsData?.allSubscriptions
```
Also, I've renamed to subscriptionsCards, what do you think?